service/watcher/configmap/cache.go

```diff
@@ -0,0 +1,213 @@
+package configmap
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/giantswarm/apiextensions/v2/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+
+	pkglabel "github.com/giantswarm/app-operator/v2/pkg/label"
+	"github.com/giantswarm/app-operator/v2/service/controller/key"
+)
+
+func (c *AppValueWatcher) buildCache(ctx context.Context) error {
+	for {
+		lo := metav1.ListOptions{
+			LabelSelector: c.selector.String(),
+		}
+
+		res, err := c.k8sClient.G8sClient().ApplicationV1alpha1().Apps("").Watch(ctx, lo)
+		if err != nil {
+			panic(err)
```

Comment:
Why do we need to panic here? 

Can we mask the error here and just panic in the top level boot function?

Reply:
Yeah, you're right, it only needs to reports error and continues, no needs to panic in here. Thanks for checking.